src/dmd/lambdacomp.d

```diff
@@ -34,6 +35,8 @@ import dmd.statement;
 import dmd.tokens;
 import dmd.visitor;
 
+enum LOG = false;
```

Comment:
Out of interest: why not `version= LOG`? This is commonly used pattern in DRuntime.

Reply:
This is the pattern used in dmd from what I could obeserve :+1: 

https://github.com/dlang/dmd/blob/master/src/dmd/mtype.d#L61
https://github.com/dlang/dmd/blob/master/src/dmd/expression.d#L67
https://github.com/dlang/dmd/blob/master/src/dmd/expressionsem.d#L73

And the list could continue.